lib/process-services-cloud/src/lib/task/task-filters/components/edit-task-filter-cloud.component.html

```diff
@@ -60,12 +60,17 @@
                         </div>
                     </mat-form-field>
                     <div fxFlex="23%" class="adf-edit-task-filter-checkbox" *ngIf="isCheckBoxType(taskFilterProperty)">
```

Comment:
can you please just use the proper flex styling in the `adf-edit-task-filter-checkbox` class? there's little reasonl using fxFlex here